### ISSUE TITLE
Get-IntuneWin32AppAssignment : missing "Target" property

### ISSUE THREAD

**Issue by ChristopheAfonso** [MSEndpointMgr/IntuneWin32App#125]
Hello,

Using cmdlet `Get-IntuneWin32AppAssignment` in version 1.4.2, new properties FilterID, FilterType etc.. are here, but the property "Target" that contains the groupId of the targeted group disappeared

module 1.4.1 : 

```
PS C:\WINDOWS\system32> Get-IntuneWin32AppAssignment -DisplayName "Winlog"

id       : 83f7b182-837e-4742-9617-d33699375fa3_1_0
intent   : required
source   : direct
sourceId : 
target   : @{@odata.type=#microsoft.graph.groupAssignmentTarget; deviceAndAppManagementAssignmentFilterId=; deviceAndAppManagementAssignmentFilterType=none; groupId=83f7b182-837e-4742-9617-d33699375fa3}
settings : @{@odata.type=#microsoft.graph.win32LobAppAssignmentSettings; notifications=showReboot; installTimeSettings=; deliveryOptimizationPriority=notConfigured; autoUpdateSettings=; restartSettings=}
```

module 1.4.2 : 

```
PS C:\Windows\System32> Get-IntuneWin32AppAssignment -DisplayName "winlog"

Type                         : #microsoft.graph.groupAssignmentTarget
AppName                      : Winlog
FilterID                     : 
FilterType                   : none
Intent                       : required
GroupMode                    : 
DeliveryOptimizationPriority : notConfigured
Notifications                : showReboot
RestartSettings              : @{gracePeriodInMinutes=1440; countdownDisplayBeforeRestartInMinutes=15; restartNotificationSnoozeDurationInMinutes=240}
InstallTimeSettings          : 
```
Is it a bug or is there a new way to retrieve the targeted groupID ?

**Comment by ChristopheAfonso** [comment]
Hello, you cannot help me ?

**Comment by NickolajA** [comment]
This was a mistake when attempting to improve the output from this function, and have been improved and fixed in version 1.4.4 that should be released shortly!
